From 66b0159d6669096c8f50114dafeeb193173fc76e Mon Sep 17 00:00:00 2001
From: Kshitij Gupta <kshitijgm@gmail.com>
Date: Sun, 11 Aug 2019 18:10:32 +0530
Subject: [PATCH 31/33] config: Include custom boardconfig a bit later

- We're trying to export MAKE_PREBUILT to soong
- Let's include it right before soong_config.mk is included

Signed-off-by: Kshitij Gupta <kshitijgm@gmail.com>
Signed-off-by: ittat <ittat@live.com>
---
 core/config.mk | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/core/config.mk b/core/config.mk
index b287edb92..e89b71d37 100644
--- a/core/config.mk
+++ b/core/config.mk
@@ -230,8 +230,6 @@ include $(BUILD_SYSTEM)/envsetup.mk
 # See envsetup.mk for a description of SCAN_EXCLUDE_DIRS
 FIND_LEAVES_EXCLUDES := $(addprefix --prune=, $(SCAN_EXCLUDE_DIRS) .repo .git)
 
-include device/oneplus/onyx/vendor/lemon/config/BoardConfigLemon.mk
-
 # The build system exposes several variables for where to find the kernel
 # headers:
 #   TARGET_DEVICE_KERNEL_HEADERS is automatically created for the current
@@ -1172,6 +1170,8 @@ dont_bother_goals := out \
     vbmetaimage-nodeps \
     product-graph dump-products
 
+include device/oneplus/onyx/vendor/lemon/config/BoardConfigLemon.mk
+
 ifeq ($(CALLED_FROM_SETUP),true)
 include $(BUILD_SYSTEM)/ninja_config.mk
 include $(BUILD_SYSTEM)/soong_config.mk
-- 
2.17.1

